deploy/deploy-azure.sh

```diff
@@ -17,7 +17,7 @@ usage() {
     echo "  -rg, --resource-group RESOURCE_GROUP       Resource group to which to make the deployment (default: \"rg-\$DEPLOYMENT_NAME\")"
     echo "  -r, --region REGION                        Region to which to make the deployment (default: \"South Central US\")"
     echo "  -a, --app-service-sku WEB_APP_SVC_SKU      SKU for the Azure App Service plan (default: \"B1\")"
-    echo "  -nq, --no-qdrant                           Don't deploy Qdrant for memory storage - Use volatile memory instead"
+    echo "  -ms, --memory-store                        Method to use to persist embeddings (default: \"AzureCognitiveSearch\")"
```

Comment:
We should list the valid options here: ""Volatile", "AzureCognitiveSearch", "Qdrant""

Reply:
Issue addressed